src/views/Rules/ListRules/index.jsx

```diff
@@ -347,14 +356,145 @@ function ListRules(props) {
       throw error;
     }
 
-    if (Object.keys(dialogRule.requiredSignoffs).length > 0) {
+    if (Object.keys(dialogRule.required_signoffs).length > 0) {
       return (await getScheduledChangeByRuleId(dialogRule.rule_id)).data
         .scheduled_changes[0];
     }
 
     return dialogRule.rule_id;
   };
 
+  const signoffDialogBody = (
+    <FormControl component="fieldset">
+      <RadioGroup
+        aria-label="Role"
+        name="role"
+        value={signoffRole}
+        onChange={handleSignoffRoleChange}>
+        {roles.map(r => (
+          <FormControlLabel key={r} value={r} label={r} control={<Radio />} />
+        ))}
```

Comment:
I can't seem to change the selection:

![Screen Shot 2019-07-30 at 9 15 58 AM](https://user-images.githubusercontent.com/3766511/62132277-bc9b7580-b2aa-11e9-9a35-32ad1050f47f.png)

Reply:
Hmm, this definitely worked at one point. I'll look into it.